utils/BUILD

```diff
@@ -77,3 +77,8 @@ cc_library(
     hdrs = ["tryte_byte_conv.h"],
     deps = ["//common:defined_error"],
 )
+
+cc_library(
+    name = "cpu_utils",
```

Comment:
The suffix `_utils` seems to be redundant since it is already located in directory `utils`. Rename it to `cpuinfo`.